### PR TITLE
Fixed fanatical.com

### PR DESCRIPTION
**Pull request by sprinkledcheese**
Fixed company logos and trustpilot rating at top.

Screenshots:
Before: 
chrome_nHg8jrT2AF

After:
firefox_a4vZAkgJiG

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1141,6 +1141,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+fanatical.com
+
+INVERT
+div.ksp:nth-child(2) > div:nth-child(1) > a:nth-child(1) > img:nth-child(1)
+
+IGNORE INLINE STYLE
+*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1150]: Using `*` Is a no-go

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1141,6 +1141,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+fanatical.com
+
+INVERT
+div.ksp:nth-child(2) > div:nth-child(1) > a:nth-child(1) > img:nth-child(1)
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1147]: `.trustpilot > .logo-container > a > img` Should do the trick if they change the DOM order.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1150]: Any alternative?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1150]: Was this just to target Steam?

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1147]: ok.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1150]: No, it changes every logo of this kind.

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -1141,6 +1141,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+fanatical.com
+
+INVERT
+.trustpilot > .logo-container > a > img
+
+IGNORE INLINE STYLE
+*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1150]: Remove this and add `.drm-container-steam` into the INVERT section that should target. That should target the steam logo

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1150]: But then I'll have to do that for other logos such as rockstar games, bethesda and a tonne of others which may come on the front page. fanatical.com has most elements in dark mode by default. So `*` doesn't have adverse effects.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1150]: I will see if they is a global solution

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1150]: Let's use `.drm-container` I see other element's that need Inline styling.

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:1150]: That fixes it.

**Pull request merged**